docker/build_scripts/build.sh

```diff
@@ -19,9 +19,8 @@ DEVTOOLS_HASH=a8ebeb4bed624700f727179e6ef771dafe47651131a00a78b342251415646acc
 # https://github.com/NixOS/patchelf/commit/2a9cefd7d637d160d12dc7946393778fa8abbc58
 PATCHELF_VERSION=2a9cefd7d637d160d12dc7946393778fa8abbc58
 PATCHELF_HASH=12da4727f09be42ae0b54878e1b8e86d85cb7a5b595731cdc1a0a170c4873c6d
-CURL_ROOT=curl-7.57.0
-# https://github.com/Homebrew/homebrew-core/blob/e3a8622111ecefe444194cade5cca3c69165e26c/Formula/curl.rb#L6
-CURL_HASH=c92fe31a348eae079121b73884065e600c533493eb50f1f6cee9c48a3f454826
+CURL_ROOT=curl_7.52.1
```

Comment:
The reason for the version downgrade is probably due to the mirror switch to Debian?

Reply:
For others who might find this PR, yes this was the reason. I'd prefer to just go to 7.58.0, but that would require switching to `ADD` in docker. I do think we should switch to that and use less bash to build this thing now that `--squash` is an option, but that's best done in `manylinux2`.